config/settings/production.py

```diff
@@ -53,9 +61,23 @@
             'format': '%(levelname)s %(asctime)s %(module)s '
                       '%(process)d %(thread)d %(message)s'
         },
+        'simple': {
+            'format':  '%(levelname)s %(asctime)s %(process)d %(thread)d %(name)s.%(module)s:%(lineno)d %(message)s',
+        },
     },
     'handlers': {
-        
+        'file' : {
+            'level' : 'INFO',
+            'class' : 'logging.FileHandler',
+            'filename' : os.path.join(filepath, env('LOG_FILE')),
```

Comment:
its very confusing..
you have a variable named LOG_DIR  and file_path in settings.py
but they never seems to be used...
instead you are using some variable from env directly...
this should be sourced from setting.py because its the only source of data for us
lets get on a call , if you are confused somewhere